src/Router.js

```diff
@@ -1,16 +1,64 @@
+import React, { useState } from 'react';
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
-import Main from 'pages/Main/Main';
+import { useFruitzState } from './FruitContext';
+import Main from './pages/Main/Main';
 import Login from './pages/Login/Login';
 import Join from './pages/Join/Join';
+import Fruit from './pages/Product/Fruit/Fruit';
+import Beverage from './pages/Product/Beverage/Beverage';
+import Detail from './pages/Product/Components/Detail/Detail';
+import Goods from './pages/Product/Goods/Goods';
+import Gifts from './pages/Product/Gifts/Gifts';
+import Aside from './components/Aside/Aside';
+import Header from './components/Header/Header';
+import Cart from './pages/Cart/Cart';
+import Contact from './pages/Contact/Contact';
+import Footer from './components/Footer/Footer';
+import MyPage from './pages/MyPage/MyPage';
+import ScrollToTop from './ScrollToTop';
 
 function Router() {
+  const token = localStorage.getItem('fruitz_user');
+  const [isLogin, setIsLogin] = useState(token ? true : false);
```

Comment:
```suggestion
  const [isLogin, setIsLogin] = useState(!!token);
```

삼항연산자 쓰지 않고도 이런식으로 truthy or falsy 값들을 true or false 형태로 변환할 수 있습니다!

Reply:
!! 에 대해서 잘 몰랐어서 다시한번 공부 했습니다 수정하였습니다!